website_parametric_snippets/docs/minimal.xml

```diff
@@ -0,0 +1,55 @@
+<openerp>
+    <data>
```

Comment:
Include those files in your manifest in the demo key, then people can play with it on runbot